client/web/compose/src/components/PageBlocks/Wrap/Card.vue

```diff
@@ -25,11 +25,11 @@
             </h5>
 
             <div
-              v-if="block.options.refreshRate >= 5 || hasMagnifyIcon"
+              v-if="magnifyOrRefresh"
               class="ml-auto"
             >
               <font-awesome-icon
-                v-if="block.options.refreshRate >= 5"
+                v-if="block.options.refreshRate >= 5 || block.options.manualRefresh"
```

Comment:
If you think about it, the button should be shown if refresh is enabled, the refreshRate will take care of the autorefresh. Otherwise, auto refresh shouldn't even work. Same for anywhere else you use this condition